pytest_client_tools/plugin.py

```diff
@@ -234,19 +234,36 @@ def save_rhc_files(request):
 @pytest.fixture
 def rhc(save_rhc_files, test_config):
     rhc = Rhc()
-    config_file = pathlib.Path("/etc/rhc/config.toml")
-    try:
-        config = toml.loads(config_file.read_text())
-    except FileNotFoundError:
+    config_file = next(
+        (
+            p
+            for p in [
+                pathlib.Path("/etc/yggdrasil/config.toml"),
```

Comment:
Should there be a dedicated fixture for yggdrasil instead? I'd be worried if our tests were doing too much background magic, it could bite us later.